src/olympia/reviewers/templates/reviewers/review.html

```diff
@@ -142,6 +139,12 @@ <h3 id="history">
     </p>
   {% endif %}
 
+  <input type="hidden" name="version_pk" value="{{ version.pk }}"/>
+  {{ form.version_pk.errors }}
+  {% if form.errors.__all__ %}
+    {{ form.errors.__all__ }}
+  {% endif %}
+
```

Comment:
Moved the error about the version here so that it's easier to spot, and added global form errors as well. That part can't be tested anymore now that I've also changed any of the existing global errors to be attached to the specific field they relate to, but it's useful for future-proofing.